src/data/icons.ts

```diff
@@ -62,7 +62,9 @@ export interface ComponentIcons {
 }
 
 interface ServiceIcons {
-  [service: string]: string;
+  [service: string]:
+    | string
+    | { service: string; sections?: { [name: string]: string } };
 }
```

Comment:
Not sure if we need to support the old syntax as the new format will always be sent by core (https://github.com/home-assistant/core/pull/124656)

Reply:
Yeah, still is discussion with core who should do it :-)